app/models/user.rb

```diff
@@ -34,7 +34,7 @@ def pending_friends
 
```

Comment:
- [ ] `pending_friends` can be defined as association with SQL query instead of processing objects in application memory (as the database was created to store and query data it is always more efficient than ruby)

```
has_many :pending_friendships, -> { where confirmed: false }, class_name: "Friendship", foreign_key: "user_id"
has_many :pending_friends, through: :pending_friendships, source: :friend

```